symfony/stimulus-bundle/2.20/manifest.json

```diff
@@ -21,7 +21,7 @@
         },
         {
             "file": "assets/app.js",
-            "content": "import './bootstrap.js';",
```

Comment:
This file is mentioned in the comment which is part of the content of the previous block adding lines in `webpack.config.js`, so you need to update the name there as well.

And the next 2 `add-lines` rule which were adding things in this `asset/bootstrap.js` file need to be updated to add them in the new file name instead.